packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -49,56 +58,112 @@ export function extractLogSafeErrorProperties(error: any) {
     return safeProps;
 }
 
-/** Type guard for ILoggingError */
-export const isILoggingError = (x: any): x is ILoggingError => typeof x?.getTelemetryProperties === "function";
+/** Metadata to annotate an error object when annotating or normalizing it */
+export interface FluidErrorAnnotations {
+    /** Telemetry props to log with the error */
+    props?: ITelemetryProperties;
```

Comment:
should probs really be optional? seems they should always be provided. if there is a case for undefined i prefer `props: ITelemetryProperties | undefined` so passing undefined is explicit

Reply:
[WF] Yeah, you might want to normalize but not have anything to add besides the error code.  And I just want people to supply the fields they want here. It's not like optional function params.